web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -72,13 +73,26 @@
 	// Progress should be reset when the current memory or asset changes.
 	$: memoryIndex, assetIndex, reset();
 
+	const handleKeyDown = (e: KeyboardEvent) => {
+		if (e.key === 'ArrowRight' && canAdvance) {
+			toNext();
+		} else if (e.key === 'ArrowLeft' && canGoBack) {
+			toPrevious();
+		} else if (e.key === 'Escape') {
+			goto(AppRoute.PHOTOS);
+		}
+	};
+
 	onMount(async () => {
 		if (!$memoryStore) {
 			const { data } = await api.assetApi.getMemoryLane({
 				timestamp: DateTime.local().startOf('day').toISO() || ''
 			});
 			$memoryStore = data;
 		}
+
+		document.addEventListener('keydown', handleKeyDown);
```

Comment:
```ts
<svelte:window on:keydown|preventDefault={handleKeydown}/>
```

Might be better?

Reply:
Not very familiar with Svelte, so I did not know about this. Thank you for the suggestion! Added.